src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+
 body {
   margin: 0;
+
+  color: $main-accent-color;
+
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+}
+
+.product-card {
+  width: 165px;
+
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  color: $main-accent-color;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    margin-bottom: 30px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    width: 166px;
```

Comment:
We avoid using these broken numbers. If you saw a number like this in the design, it must certainly comes from another styling, like `width: 100%` or something like that